policyengine_us/parameters/gov/states/nm/tax/income/rebates/low_income/amount/five_exemptions.yaml

```
@@ -4,78 +4,97 @@ brackets:
       2021-01-01: 0
     amount:
       2021-01-01: 455
+      2023-01-01: 492
   - threshold: 
       2021-01-01: 1_001
```

Then use `calc(..., right=True)` in the formula. This better mirrors the law.
```suggestion
      2021-01-01: 1_000
```